torch/utils/_sympy/value_ranges.py

```diff
@@ -97,11 +98,11 @@ def sympy_generic_le(lower, upper):
         return not (lower and not upper)
 
 
-def vr_is_bool(vr: ValueRanges[_T]) -> TypeIs[ValueRanges[SympyBoolean]]:
+def vr_is_bool(vr: ValueRanges[_T]) -> TypeGuard[ValueRanges[SympyBoolean]]:
```

Comment:
No, these should be TypeIs imported from typing extensions

Reply:
What are the errors they are causing out of curisosity?

Reply:
The problem is that TypeIs requires the resulting type to be a strict narrowing of the original type.  This is not so it's getting an [narrowed-type-not-subtype] error.